config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   root "root#index"
+  get '/fullArticle/:id', to: 'root#index'
```

Comment:
I think it should be `/article/:id` - which is created behind the scenes with the `resources` keyword below.